stackint/asm/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
The `stackint/asm` directory mostly contains code taken (and modified) from the math/big library. Because of this, the golang license has been included in the directory.